### ISSUE TITLE
BERTMHC: improved MHC–peptide class II interaction prediction with transformer and multiple instance learning 

### ISSUE THREAD

**Issue by ghuijo** [One-Day-One-Paper/One-Day-One-Paper#322]
*MHC dataset 관련*

* 인코딩 쪽
  <image src="https://user-images.githubusercontent.com/67627471/200176481-42132692-e8f1-42d7-bf4e-ca66eb0b9a6d.png" width=50% height=50%>

<image src="https://user-images.githubusercontent.com/67627471/200176669-6c17f296-8c29-4507-9c48-3fc3a11600b2.png" width=40% height=40%>

* use the pretrained bidirectional encoder representations from transformers neural network from the TAPE repository
to model the input amino acid sequences
* self attention: bidirectional next-token prediction + masked-token prediction
* trained two BERT models:
  * MHC class II binding
  * presentation prediction
* MA deconvolution with MIL
  * In the MA presentation data, each peptide is associated with a bag of alleles
  * The bag is labeled as positive if at least one of the alleles presented the peptide
  * otherwise, the bag is labeled as negative.

<image src="https://user-images.githubusercontent.com/67627471/200176687-2c9aaaf9-4024-49d4-97f6-02ab9df85af9.png" width=60% height=60%>
